src/Statistics/include/Statistics/Histogram.h

```diff
@@ -13,6 +13,13 @@
 
 namespace orbit_statistics {
 
+// Represents the inclusive range the user has selected on the HistogramWidget.
+// Advise on better placement is welcome
+struct HistogramSelectionRange {
+  uint64_t start;
```

Comment:
Isn't it more like a "min" and "max" value?

Reply:
Makes sense. But do you think this is the right place for this structure to be defined? It's totally unrelated to the rest of Statistics/

Reply:
Its probably not the best place, but I guess would be still fine. I don't have a better idea right now. We can move it later, I guess (unless someone else has a better idea).